src/pages/edit/ItemSelector/__tests__/__snapshots__/ContentMenuItem.spec.js.snap

```diff
@@ -3,14 +3,17 @@
 exports[`ContentMenuItem has a LaunchLink when url is provided 1`] = `
 <div>
   <li
-    class="jsx-1930534478 "
+    class="jsx-3192368000 "
```

Comment:
Maybe this test needs to be changed so we don't get these diffs when the ui version is upgraded.